Mafqood/config/initializers/assets.rb

```diff
@@ -9,6 +9,5 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w(grid.js modernizr.custom.js component.css default.css)
-Rails.application.config.assets.precompile += %w(finding_grid.js modernizr.custom.js component.css default.css)
-Rails.application.config.assets.precompile += %w(missing_grid.js modernizr.custom.js component.css default.css)
+Rails.application.config.assets.precompile += %w(modernizr.js finding_grid.js component.css default.css)
+Rails.application.config.assets.precompile += %w(modernizr.js missing_grid.js component.css default.css)
```

Comment:
@mohamed-mehany you can merge those 2 lines to:
Rails.application.config.assets.precompile += %w(modernizr.js missing_grid.js finding_grid.js component.css default.css)